corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -136,16 +136,27 @@
         </div>
 
         <div class="tab-pane" id="groups">
+        {% if needs_to_downgrade_locations %}
+            <div>
+                <p>
+                    {% trans "This project no longer has access to the Organization feature." %}
+                    <a href="{% url "downgrade_locations" domain %}">
+                        {% trans "Click here to fix this." %}
```

Comment:
@orangejenny 

Felt a bit confusing to make a change for the entire project from within the mobile worker edit page.   Amelia's rationale about forcing someone to confront locations and the impact of their change is correct.